src/ManualSetupTotpController.js

```diff
@@ -33,44 +34,54 @@ export default FormController.extend({
   Form: {
     title: function() {
       const factorName = FactorUtil.getFactorLabel(this.model.get('__provider__'), this.model.get('__factorType__'));
-
       return loc('enroll.totp.title', 'login', [factorName]);
     },
-    subtitle: _.partial(loc, 'enroll.totp.cannotScanBarcode', 'login'),
```

Comment:
Again - isn't this a breaking change? Customizations for this label will now no longer be present for legacy pipeline users.